include/envoy/stream_info/stream_info.h

```diff
@@ -78,6 +78,8 @@ enum ResponseFlag {
   ResponseFromCacheFilter = 0x100000,
   // Filter config was not received within the permitted warming deadline.
   NoFilterConfigFound = 0x200000,
+  // Request hit the max_downstream_connection_duration, triggering a 408.
```

Comment:
I think you copy-pasted a comment for HTTP :-)

Maybe

// Request or connection exceeded the downstream connection duration
?

It'd also be great to test the flag is set in your unit test.